maintainers/maintainer-list.nix

```diff
@@ -4323,6 +4323,16 @@
     githubId = 54999;
     name = "Ariel Nunez";
   };
+  irenes = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add irenes`` and sort it alphabetically.

Reply:
done; please take a look.